INSTALL.md

```diff
@@ -229,6 +230,56 @@ And that's it!  Recompilation should work normally now, though you will
 potentially need to restart your computer, or at least the running X
 session, first.
 
+### Build using cabal-install
+
+#### Install cabal-install
+
+The easiest way to get [cabal-install] is probably via your system's package
+manager:
+
+``` shell
```

Comment:
I wonder if we could get nicer syntax highlighting via `console` here (and in other places) or if that doesn't play nicely with the comments

Reply:
It definitely does make the `$ stack init` block easier on the eyes: https://github.com/xmonad/xmonad/blob/62069e8af70de7793e7541eaf4c6381521469456/INSTALL.md#create-a-new-project vs https://github.com/xmonad/xmonad/blob/11e41b98d151662627efba16cc4d3eb00ae2d8b6%5E%5E/INSTALL.md#create-a-new-project
Most other blocks look the same. I had to add leading `> ` to [multiline commands in dependencies](https://github.com/xmonad/xmonad/blob/62069e8af70de7793e7541eaf4c6381521469456/INSTALL.md#dependencies) because the `console` highlighter would otherwise [highlight the continuation lines as output](https://github.com/xmonad/xmonad/blob/6b3a0048f7719124c8b42772ef7d3acded99ac03/INSTALL.md#dependencies).

Reply:
Overall it's probably better so I'll wait a couple hours for you to see it and then rebase and merge. Pls ping me on IRC or here then.